languages/java/exercises/concept/inheritance/src/test/java/FighterTest.java

```diff
@@ -0,0 +1,55 @@
+import org.junit.Ignore;
+import org.junit.Test;
+
+import static junit.framework.TestCase.*;
+
+public class FighterTest {
+
+    @Ignore
```

Comment:
Usually the first test should not be ignored

Reply:
Oh okay, I'll change that.